### PR TITLE
Add clarifying unit of measurement

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?
- Add unit of measurement for Logs batch wait time.

### Motivation
Customer feedback on documentation is that the lack of unit of measurement is unclear
https://datadoghq.atlassian.net/browse/DOCS-8347
Confirmation [slack](https://dd.slack.com/archives/C046YU1V0SK/p1720013949388949)

### Possible Drawbacks / Trade-offs
This only adds a unit of measurement for the one parameter, not all parameters state the unit of measurement which looks inconsistent.

### CODE REVIEW

File: pkg/config/config_template.yaml [hunk 1]
```diff
@@ -973,6 +973,7 @@ api_key:
   ## @param batch_wait - integer - optional - default: 5
   ## @env DD_LOGS_CONFIG_BATCH_WAIT - integer - optional - default: 5
   ## The maximum time the Datadog Agent waits to fill each batch of logs before sending.
+  ## The maximum batch wait in seconds.
```
Comment (pgimalac) [inline pkg/config/config_template.yaml:976]: What do you think of 

```suggestion
  ## The maximum time (in seconds) the Datadog Agent waits to fill each batch of logs before sending.
```
or
```suggestion
  ## The maximum time the Datadog Agent waits to fill each batch of logs before sending (in seconds).
```
?

Comment (estherk15) [reply pkg/config/config_template.yaml]: Thank you @pgimalac, that's a much better solution!

**Review by urseberry — APPROVED** [review]

**Review by pgimalac — APPROVED** [review]

**Pull request merged**